### ISSUE TITLE
Error: Invalid Attribute Value Match when topic name has a dots

### ISSUE THREAD

**Issue by allebacco** [conduktor/terraform-provider-conduktor#8]
When the name of a topic contains dots, the validation of the Terraform resource fails.

The following resource

```hcl
resource "conduktor_user_v2" "test_user" {
  name = "test.user@example.com"
  spec {
    firstname   = "Test"
    lastname    = "User"
    permissions = [
        {
            resource_type = "TOPIC"
            name          = "valid.topic.name.with.dots.0"
            cluster       = "my-cluster"
            pattern_type  = "LITERAL"
            permissions   = ["topicViewConfig", "topicConsume", "topicProduce"]
        },
    ]
  }
}
```

create the following error:

```text
Error: Invalid Attribute Value Match
│ 
│   with conduktor_user_v2.test_user,
│   on users.tf line 16, in resource "conduktor_user_v2" "test_user":
│   16:   spec {
│ 
│ Attribute spec.permissions[0].name value must match regular expression '^[0-9a-z\_\-\*]+$', got: valid.topic.name.with.dots.0
```

I think that the problem is due to the following line: also dots should be accepted in the name.

https://github.com/conduktor/terraform-provider-conduktor/blob/08e35f8f9c07e1d4119d369604794f14b977cbd5/internal/schema/resource_user_v2/user_v2_resource_gen.go#L71

**Comment by qboileau** [comment]
Hi @allebacco 

thanks for reporting this issue. 

It seem to be a bug caused by the simplification of the schema structure for permissions that flatten what is in reallity an ADT on Console API side. 
Because of that this regex check on `name` come from Cluster permissions that should not be here for Topic permissions. 

We will remove this regex check on provider and let the API side handle the potential errors until we came up with a better permission representation in provider schema.

**Comment by qboileau** [comment]
Released in [v0.1.1](https://github.com/conduktor/terraform-provider-conduktor/releases/tag/v0.1.1) sorry for the delay.
